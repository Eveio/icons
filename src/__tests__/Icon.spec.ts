import IconAccreditation from '../icons/IconAccreditation.vue'
import { mount } from '@vue/test-utils'

describe('Icon', () => {
  it('should render', () => expect(mount(IconAccreditation).html()).toMatchSnapshot())

  it('should pass classes through', () => {
    const wrapper = mount(IconAccreditation, {
      propsData: {
        class: 'icon-xs'
      }
    })
    expect(wrapper.classes()).toContain('icon-xs')
  })
})
