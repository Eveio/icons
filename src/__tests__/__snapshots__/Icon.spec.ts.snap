// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should render 1`] = `
"<svg viewBox=\\"0 0 20 20\\" fill=\\"none\\" xmlns=\\"https://www.w3.org/2000/svg\\" class=\\"icon\\">
  <g id=\\"accreditation\\" fill=\\"currentColor\\">
    <path d=\\"M17.037 2.917h-4.074c0-.774-.312-1.516-.868-2.063A2.987 2.987 0 0010 0c-.786 0-1.54.307-2.095.854a2.894 2.894 0 00-.868 2.063H2.963c-.786 0-1.54.307-2.095.854A2.894 2.894 0 000 5.833v8.75c0 .774.312 1.516.868 2.063.555.547 1.31.854 2.095.854h14.074c.786 0 1.54-.307 2.095-.854A2.894 2.894 0 0020 14.583v-8.75c0-.773-.312-1.515-.868-2.062a2.987 2.987 0 00-2.095-.854zM10 1.458c.393 0 .77.154 1.048.427.277.274.434.645.434 1.032H8.518c0-.387.156-.758.433-1.032A1.494 1.494 0 0110 1.458zm-.37 14.584v-.73c0-.773.312-1.515.867-2.062a2.987 2.987 0 012.096-.854c.785 0 1.54.307 2.095.854.555.547.868 1.289.868 2.063v.729H9.63zm1.481-6.563c0-.288.087-.57.25-.81.163-.24.394-.427.665-.537a1.503 1.503 0 011.614.316 1.437 1.437 0 01.321 1.59 1.463 1.463 0 01-.545.654 1.498 1.498 0 01-1.871-.182 1.447 1.447 0 01-.434-1.03zm7.407 5.104c0 .387-.156.758-.433 1.031a1.493 1.493 0 01-1.048.428v-.73c0-.782-.213-1.55-.618-2.225a4.413 4.413 0 00-1.682-1.606 2.9 2.9 0 00.771-1.495 2.875 2.875 0 00-.192-1.666 2.928 2.928 0 00-1.092-1.287 2.997 2.997 0 00-3.263 0A2.928 2.928 0 009.87 8.32a2.875 2.875 0 00-.192 1.666 2.9 2.9 0 00.771 1.495 4.413 4.413 0 00-1.682 1.606 4.323 4.323 0 00-.618 2.226v.729H2.963c-.393 0-.77-.154-1.048-.428a1.447 1.447 0 01-.434-1.03v-8.75c0-.387.157-.758.434-1.032a1.493 1.493 0 011.048-.427h14.074c.393 0 .77.154 1.048.427.277.274.433.645.433 1.031v8.75z\\"></path>
    <path d=\\"M3.704 9.167h2.963a.702.702 0 00.523-.244.889.889 0 00.217-.59.889.889 0 00-.217-.589.702.702 0 00-.523-.244H3.704a.702.702 0 00-.524.244.889.889 0 00-.217.59c0 .22.078.432.217.589a.702.702 0 00.524.244zM3.704 10.417a.702.702 0 00-.524.244.89.89 0 00-.217.59c0 .22.078.432.217.588a.702.702 0 00.524.244h2.963a.702.702 0 00.523-.244.889.889 0 00.217-.589.89.89 0 00-.217-.59.702.702 0 00-.523-.243H3.704z\\"></path>
  </g>
</svg>"
`;
